crypto/src/address.rs

```diff
@@ -64,3 +67,44 @@ impl Address {
         &self.ck_d
     }
 }
+
+impl std::fmt::Display for Address {
+    fn fmt(&self, f: &mut std::fmt::Formatter) -> std::fmt::Result {
+        let mut addr_content = std::io::Cursor::new(Vec::new());
+        addr_content
+            .write_all(&self.diversifier().0)
+            .expect("can write diversifier into vec");
+        addr_content
+            .write_all(&self.transmission_key().0)
+            .expect("can write transmission key into vec");
+        addr_content
+            .write_all(&self.clue_key().0)
+            .expect("can write clue key into vec");
+
+        // Mainnet "pm" prefix, testnet "pt" prefix
+        let human_part = "pt";
```

Comment:
Cosmos uses `cosmos` which is a bit longer but I think reads more nicely -- WDYT of `penumbra` for the human-readable part? The addresses are long enough already that I don't think a few extra characters is going to make a big difference.

For the testnets, I think it would be useful to encode the testnet number in the address, like `penumbra-tn001-`, `penumbra-tn002-`, etc (not sure about the exact format, but 3 digits seems good, and having a trailing dash separates the number from the bech32 `1` separator).

Reply:
Yeah it seems like underscores have better double-click to select behavior so going with that (for mainnet we wont have the suffix so can just do `penumbra` and this won't be an issue)